test/unit/chain.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @module test/unit/chain
+ */
+
+'use strict';
+
+require('chai').should();
+
+const Account = require('core/account');
+const chain = require('core/chain');
+
+const SECRET_KEYS = [
+    Buffer.from('557dce58018cf502a32b9b7723024805399350d006a4f71c3b9f489f7085cb50', 'hex'),
+    Buffer.from('9d22d682fd4ad16b0f6e21eddfe46d7231fff8670507a184787f74d6538ab0bc', 'hex')
+];
+
+describe('Chain', () => {
+    let account = {};
+    let delegate = {};
+    let delegates = [];
+
+    before('create', () => {
+        account = Account(SECRET_KEYS[0]);
+        delegate = Account(SECRET_KEYS[1]);
+    });
+
+    it('get account balance', async () => {
+        const balance = await chain.getBalance(account.getStrAddress());
+
+        balance.should.be.a('number');
+
+        console.log('Balance: ', balance);
```

Comment:
If you are passing arguments using comma separation extra space will be added automatically.
```suggestion
        console.log('Balance:', balance);
```